src/functions/getJournal/framework/handler.ts

```diff
@@ -11,8 +11,8 @@ export async function handler(event: APIGatewayProxyEvent, fnCtx: Context) {
     return createResponse('No staffNumber provided', HttpStatus.BAD_REQUEST);
   }
 
-  const employeeId = getEmployeeIdFromToken(event.headers.Authorization);
   if (process.env.EMPLOYEE_ID_VERIFICATION_DISABLED !== 'true') {
+    const employeeId = getEmployeeIdFromToken(event.headers.Authorization);
```

Comment:
Nice spot!

Reply:
Wouldn't have noticed if it weren't for the fact that the integration test was failing due to not providing a JWT!